tests/__init__.py

```diff
@@ -29,17 +29,16 @@
 # to their authors.
 
 def find_empty_redis_database():
-    """Connect to a random Redis database.
+    """Connect to a fake Redis database.
 
-    Tries to connect a random Redis database (starting on 8) and
+    Tries to connect a FakeRedis database and
     will use/connect it when no keys are stored in there.
     """
-    for db in range(8, 16):
-        conn = StrictRedis(db=db)
-        empty = len(conn.keys('*')) == 0
-        if empty:
-            return conn
-    assert False, "No empty Redis database found to run tests in."
+    conn = FakeStrictRedis()
+    empty = len(conn.keys('*')) == 0
```

Comment:
All these code can be removed because it was only needed when a Redis server was in use. It tried to avoid data corruption in existing databases.